src/routes/choice-exercises/ArticleChoiceExerciseScreen.tsx

```diff
@@ -24,23 +24,16 @@ const ArticleChoiceExerciseScreen = ({
     return ARTICLES.filter(article => article.id !== 0).map(article => ({ article, word: document.word }))
   }
 
-  const onExerciseFinished = (results: DocumentResultType[]): void => {
-    navigation.navigate('InitialSummary', {
-      result: { discipline: { ...route.params.discipline }, results, exercise: ExerciseKeys.articleChoiceExercise }
-    })
-  }
-
   if (documents === null || loading) {
     return null
   }
-
   return (
     <SingleChoiceExercise
-      documents={documents}
+      data={documents}
       documentToAnswers={documentToAnswers}
-      onExerciseFinished={onExerciseFinished}
```

Comment:
Why is this not needed anymore?

Reply:
since the function is identical to the one in WordChoiceExerciseScreen, its sufficient to pass the exerciseKey and move the function to `SingleChoiceExercise`